### PR TITLE
[CIFuzz] support at PR

### PR DESCRIPTION
**Pull request by 0x34d**
Integrating [CIFuzz](https://google.github.io/oss-fuzz/getting-started/continuous-integration) for pull requests to identify potential bugs prior to merging the changes.

### CODE REVIEW

**Comment by LBeernaertProton** [comment]
Could you write the fuzz  extra step into `workflows/pull-request.yml` instead?

**Comment by 0x34d** [comment]
> Could you write the fuzz extra step into `workflows/pull-request.yml` instead?

Do you mean `.github/workflows/cifuzz.yml` to `.github/workflows/pull-request.yml`

**Comment by LBeernaertProton** [comment]
> > Could you write the fuzz extra step into `workflows/pull-request.yml` instead?
> 
> Do you mean `.github/workflows/cifuzz.yml` to `.github/workflows/pull-request.yml`

Yes.

**Comment by 0x34d** [comment]
> > > Could you write the fuzz extra step into `workflows/pull-request.yml` instead?
> > 
> > 
> > Do you mean `.github/workflows/cifuzz.yml` to `.github/workflows/pull-request.yml`
> 
> Yes. The fuzz job did not appear in your MR, I assume there is an issue with the workflow. In any case, it would be nicer if everything was organized in that file.

There was an error `bad indentation of a mapping entry`

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -47,3 +47,25 @@ jobs:
       - name: Run tests with race check
         if: runner.os != 'Windows'
         run: go test -race -v ./tests
+
+  Fuzzing:
```
Comment (0x34d) [inline .github/workflows/pull-request.yml:51]: Should I remove it? or let it be.

Comment (LBeernaertProton) [reply .github/workflows/pull-request.yml:51]: This is fine.

**Review by LBeernaertProton — APPROVED** [review]

**Pull request merged**